pkg/util/util_test.go

```diff
@@ -20,6 +19,10 @@ var (
 	TestArchivePath string
 	// TestArchiveExtractDir is the directory in the archive to extract
 	TestArchiveExtractDir = "wordpress-0.4.0/"
+
+	// The image here can be any image, it just has to exist so it can be used for labels, etc.
+	TestRouterImage = "drud/nginx-proxy"
+	TestRouterTag   = "test-do-not-delete"
```

Comment:
busybox should work for this test. It does not require a specific image, it simply needs a container.